tools/please_go/please_go.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"log"
+	"os"
+
+	"github.com/peterebden/go-cli-init/v4/flags"
+
+	"github.com/thought-machine/please/tools/please_go/install"
+)
+
+var opts = struct {
+	Usage string
+
+	PleaseGoInstall struct {
+		SrcRoot      string `short:"r" long:"src_root" description:"The src root of the module to inspect" default:"."`
+		ModuleName   string `short:"n" long:"module_name" description:"The name of the module"`
```

Comment:
are any of these required?

Reply:
Yeah all of them except SrcRoot. Ah I see they're optional by default. Let me fix that.